insights/parsers/kpatch_patches.py

```diff
@@ -0,0 +1,40 @@
+"""
+KpatchPatches - report locally stored kpatch patches
+====================================================
+
+This parser creates a list of the module names of locally
+stored kpatch modules. If no modules are installed, the
+list will be empty.
+
+"""
+
+from .. import parser, Parser
+from insights.specs import Specs
+import re
+
+
+@parser(Specs.kpatch_patch_files)
+class KpatchPatches(Parser):
+    """
+    A parser for getting locally stored kpatch-patch modules
+    """
+    __re_cannot_access = re.compile(r"No such file or directory$")
```

Comment:
By leveraging the `CommandParser` this check can be saved.

Reply:
This check is removed in the latest patch